net/local/local_sendmsg.c

```diff
@@ -125,7 +125,7 @@ static int local_sendctl(FAR struct local_conn_s *conn,
               goto fail;
             }
 
-          ret = file_dup2(filep, filep2);
+          ret = file_dup2(filep, filep2); /* TODO: What about this ? */
```

Comment:
what's problem here to add this comment?

Reply:
Explained above. I don't understand what these drivers are trying to do with these dup/dup2 calls, so someone has to explain it to me.

Reply:
@anchao can you please help, it looks like you wrote local_recvctl and local_sendctl and they use file_dup and file_dup2. Could you tell me if they need to be changed somehow to fit into this new fd/file implementation ?

Reply:
SCM (Socket-Control-Message) needs to provide the ability to pass file descriptors. 
https://man7.org/linux/man-pages/man3/cmsg.3.html
There will be no side effect if file_dup behaves as expected.

Reply:
Thanks, I'll take a look at that spec. It helps a lot.

Reply:
I think it should work like before; file_dup creates a new filep for fd2 and and copies the data from filep1 -> filep2, just like before. I presume this is the expected behavior.